Security/Firewall/OAuthListener.php

```diff
@@ -29,9 +29,9 @@
 class OAuthListener implements ListenerInterface
 {
     /**
-     * @var \Symfony\Component\Security\Core\SecurityContextInterface
+     * @var \Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface
      */
-    protected $securityContext;
+    protected $tokenStorage;
```

Comment:
renaming protected properties is a BC break. I would keep the old name here for BC